TeamCode/src/main/java/org/firstinspires/ftc/teamcode/subsystems/claw/ClawCommands.java

```diff
@@ -0,0 +1,15 @@
+package org.firstinspires.ftc.teamcode.subsystems.claw;
+
+import com.arcrobotics.ftclib.command.InstantCommand;
+
+import org.firstinspires.ftc.teamcode.Robot;
+
+public class ClawCommands {
+
+    public static final InstantCommand openBothFingers = new InstantCommand(() -> Robot.getInstance().CLAW.openBothClaws());
+    public static final InstantCommand closeBothFingers = new InstantCommand(() -> Robot.getInstance().CLAW.closeBothClaws());
+    public static final InstantCommand toggleRightFinger = new InstantCommand(() -> Robot.getInstance().CLAW.toggleRightClaw());
+    public static final InstantCommand toggleLeftFinger = new InstantCommand(() -> Robot.getInstance().CLAW.toggleLeftClaw());
+    public static final InstantCommand switchClaws = new InstantCommand(() -> Robot.getInstance().CLAW.switchClaws());
```

Comment:
consider changing the type of these to `Command`.

Reply:
sincerely, קפוץ לי

![Robot v32fsgdhjfkl](https://github.com/GreenBlitz/FTC-CENTERSTAGE/assets/108526263/1eb015e4-8f95-41df-9c46-97e080ef1f73)

Reply:
טוב נו